backends-clickhouse/src/test/scala/org/apache/gluten/execution/GlutenClickhouseFunctionSuite.scala

```diff
@@ -66,6 +66,11 @@ class GlutenClickhouseFunctionSuite extends GlutenClickHouseTPCHAbstractSuite {
       // TODO: support default ANSI policy
       .set("spark.sql.storeAssignmentPolicy", "legacy")
       .set("spark.sql.warehouse.dir", warehouse)
+      .set(
```

Comment:
set this config in the following ut case ? otherwise it can impact the other cases.

Reply:
This suite is for test functions, disable unfold constant functions should be OK